app/models/favorite_project.rb

```diff
@@ -3,4 +3,8 @@ class FavoriteProject < ActiveRecord::Base
   belongs_to :user
 
   validates_uniqueness_of :project_id, :scope => :user_id
+
+  def self.created_between(start_date, end_date)
+    where('created_at >= ? AND created_at <= ?', start_date, end_date)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.